lib/rover/data_frame.rb

```diff
@@ -436,7 +436,8 @@ def plot(x = nil, y = nil, type: nil)
           .mark(type: type, tooltip: true, interpolate: "cardinal", point: {size: 60})
           .encoding(
             x: {field: x, type: x_type, scale: scale},
-            y: {field: y, type: "quantitative"}
+            y: {field: y, type: "quantitative"},
+            color: {field: group}
```

Comment:
Sorry forgot to mention last time: we should only add the `color` option if `group` isn't `nil`.

Reply:
I think that it still works the same way regardless of adding a ternary operator.